src/components/product-card/HorizontalCard/ProductCardHorizontal.jsx

```diff
@@ -44,49 +44,153 @@ const ProductCardHorizontal = ({ item }) => {
     }
   }
 
+  async function removeFromWishlist(product) {
+    try {
+      const responseWishlist = await axios.get(`/api/user/wishlist`, {
+        headers: {
+          authorization: loginToken,
+        },
+      });
+      if (
+        responseWishlist.data.wishlist.find((item) => item._id === product._id)
```

Comment:
Same here, you can reduce one api call...

Reply:
I will fix these changes in next feature commit